### DISCUSSION TITLE
Feature: Lock screen network flyout

### DISCUSSION THREAD

**Discussion by abbzie** [valinet/ExplorerPatcher#2139] [Ideas]
If it is possible, use the lock screen network flyout instead of the one currently under "Windows 10". the current one is quite buggy, such as the buttons at the bottom not working properly and having cut off text.

**Comment by vippium** [comment]
You can fix it manually. Go to  **C:\Windows\SystemApps\ShellExperienceHost_cw5n1h2txyewy** and rename dxgi.dll and the issue will be fixed

**Comment by abbzie** [comment]
> You can fix it manually. Go to **C:\Windows\SystemApps\ShellExperienceHost_cw5n1h2txyewy** and rename dxgi.dll and the issue will be fixed

What do i rename it to?

**Comment by vippium** [comment]
Anything like dxgi. dllqqw or something lie that
